src/property-filter/styles.scss

```diff
@@ -39,7 +39,6 @@
 
 .token-editor {
   margin: awsui.$space-xxs;
-  min-width: 300px;
```

Comment:
This fixes responsiveness.

<img width="378" alt="Screenshot 2022-09-15 at 11 18 23" src="https://user-images.githubusercontent.com/20790937/190366253-d6e4e997-a2fd-4344-a41c-56fa2f3c5854.png">